ValueError has no attribute message

`tnefparse/cmdline.py:68: error: "ValueError" has no attribute "message"`

This is Python 2 code.

The fix is easy -> use `str(e)` instead. The test is a bit more elaborate - when there is no suitable test file.

This bug was uncovered only now, as the line has no test coverage.

see https://github.com/koodaamo/tnefparse/blob/a9668efd84ecd0452436517e675bdf9654c2e8ba/tnefparse/cmdline.py#L64-L67

--- a/tests/test_cmdline.py
+++ b/tests/test_cmdline.py
@@ -1,6 +1,12 @@
 import json
 import shutil
+import sys
 import tempfile
+from unittest.mock import patch
+
+import pytest
+
+import tnefparse.cmdline
 
 
 def test_cmdline_overview(script_runner):
@@ -64,3 +70,15 @@ def test_dump(script_runner):
     dump = json.loads(ret.stdout)
     assert sorted(list(dump.keys())) == ['attachments', 'attributes', 'extended_attributes']
     assert len(dump['attachments']) == 2
+
+
+@patch.object(sys, 'argv', ['tnefparse', 'tests/examples/two-files.tnef'])
+@patch.object(tnefparse.cmdline, "TNEF")
+def test_handle_value_errors(mocked_TNEF):
+    """Make sure e.g. invalid TNEF signatures are handled"""
+    # If there is a suitable TNEF file with an e.g. invalid TNEF signature,
+    # this test should be updated.
+    mocked_TNEF.side_effect = ValueError("Value not allowed.")
+    with pytest.raises(SystemExit):
+        from tnefparse.cmdline import tnefparse
+        tnefparse()

--- a/tnefparse/cmdline.py
+++ b/tnefparse/cmdline.py
@@ -64,7 +64,7 @@ def tnefparse():
         try:
             t = TNEF(tfp.read(), do_checksum=args.checksum)
         except ValueError as exc:
-            sys.exit(exc.message)
+            sys.exit(str(exc))
         if args.overview:
             print("\nOverview of %s: \n" % tfp.name)
 

